orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationChatResponse.java

```diff
@@ -62,12 +62,12 @@ public List<ChatMessage> getAllMessages() {
   }
 
   /**
-   * Get current choice.
+   * Get the LLM response. Useful for accessing the finish reason or further data like logprobs.
    *
-   * @return The current choice.
+   * @return The (first, in case of multiple) {@link LLMChoice}.
    */
   @Nonnull
-  private LLMChoice getCurrentChoice() {
+  public LLMChoice getCurrentChoice() {
```

Comment:
Where does the "current" word come from?

Reply:
There might be previous LLM results in `allMessages` (e.g. in the chat scenario). So this represents the last i.e. most recent LLM response.

But I think there is no strong reason for "current", we could also just call it `getLlmChoice`

Reply:
I was hoping you can cite a source that uses "current" too. Then I wouldn't have a problem with that.

Otherwise, technically it returns the "first" item. Or presumably(?) the most "recent" item. Not sure how the ordering is part of the spec contract.

Currently, with my limited understanding, the safest approach would be just calling it `get[Llm]Choice`.

Reply:
Renamed to `getChoice`